### PR TITLE
add layer property for automatic shapes

### LINKED ISSUE #715: Detected shapes are not part of the selected layer
### What happened?

A bug happened!

### Version

v2.1.1-rc.1

### Platform

Android

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by MrGumpIT**
Closes #715

### CODE REVIEW

File: app/lib/handlers/pen.dart [hunk 1]
```diff
@@ -180,6 +180,8 @@ class PenHandler extends Handler<PenTool> with ColoredHandler {
     // Create recognizeUnistroke
     final recognized = recognizeUnistroke(points);
     final element = elements[pointer];
+    final state = context.getDocumentBloc().state as DocumentLoadSuccess;
```
Comment (CodeDoctorDE) [inline app/lib/handlers/pen.dart:183]: you can just use context.getState(), I had made a shortcut for this

**Review by CodeDoctorDE — COMMENTED** [review]
Looks good otherwise

**Comment by Luciogi** [comment]
minor fix: replace `propert` with `property` in commit message

**Comment by CodeDoctorDE** [comment]
Thanks for contributing!

**Pull request merged**